week-1-practise/level-2/1-tax.js

```diff
@@ -5,7 +5,9 @@
   Sales tax is 20% of the price of the product
 */
 
-function calculateSalesTax() {}
```

Comment:
@Atayo  Its a good work!, but you don't need to write the function declaration twice you need to delete line number 8.